### PR TITLE
Modify the opcode based on php 7.4.0.

### PR DESCRIPTION
**Pull request by jimmyleeeeee**
Modify the opcode based on php 7.4.0, as the codes of some  insttruction has been changed.

### CODE REVIEW

File: srm_oparray.c [hunk 1]
```diff
@@ -110,7 +124,11 @@ static const op_usage opcodes[] = {
 #else
 	/*  49 */	{ "CHECK_VAR", OP1_USED },
 	/*  50 */	{ "SEND_VAR_NO_REF_EX", ALL_USED },
-	/*  51 */	{ "MAKE_REF", RES_USED| OP1_USED },
+#if PHP_VERSION_ID >= 70400
+	/*  51 */	{ "CAST", ALL_USED | EXT_VAL },
+#else
+	/*	51 */	{ "MAKE_REF", RES_USED| OP1_USED},
```
Comment (derickr) [inline srm_oparray.c]: Can you fix the white space here please?

Comment (jimmyleeeeee) [reply srm_oparray.c]: OK. I will fix it soon.

Comment (jimmyleeeeee) [reply srm_oparray.c]: I am sorry that I used tab here.  And I have fix it by replacing tab with white spaces. Please review it again.

**Comment by jimmyleeeeee** [comment]
I have merged the code to master branch after being reviewed. Close this pull request.

**Pull request closed without merging**